server/controllers/api/editFlight.js

```diff
@@ -1,24 +1,47 @@
-
 import models from '../../database/models/index';
 import sendSmsNotifications from './sendSms';
 
-export default(req, res, next) => {
+export default (req, res, next) => {
   const {
-    gateNo, status, terminalNo, departureTime,
+    gateNo,
+    status,
+    terminalNo,
+    departureTime,
   } = req.body;
-  const { flightId: flightNo } = req.params;
+  const {
+    flightId: flightNo,
+  } = req.params;
 
   models.Flights
     .update({
-      gate: gateNo, status, terminal_no: terminalNo, departure_time: departureTime,
-    }, { where: { flight_no: flightNo }, returning: true, plain: true })
```

Comment:
Please separate your files, make a query with this `models.Flights.update` and call it back here, no calling models inside your controllers.